bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RegionAwareEnsemblePlacementPolicy.java

```diff
@@ -500,19 +504,26 @@ protected BookieNode replaceFromRack(BookieNode bookieNodeToReplace,
     }
 
     @Override
-    public final List<Integer> reorderReadSequence(ArrayList<BookieSocketAddress> ensemble, List<Integer> writeSet, Map<BookieSocketAddress, Long> bookieFailureHistory) {
```

Comment:
It is worth adding a BC test comparing the logic here with the logic in 4.5.0. they should be the same.

Reply:
The tests that are already there effectively test B.C. as they put a write set through a reorder and check if it matches a set value.